qemu/tests/virtio_port_hotplug.py

```diff
@@ -71,8 +72,14 @@ def run(test, params, env):
                 error_context.context("Load  module %s" % module, logging.info)
                 session.cmd("modprobe %s" % module)
         session.close()
+    logging.info("Kill the first serial process on host")
+    host_script = params['host_script']
+    result = process.system('pkill -f %s' % host_script, shell=True)
```

Comment:
If the original process is killed, the result is non 0.So, better to check if it existes.

Reply:
> If the original process finished, the result is non 0.So, better to check if it existes.

Thanks and update.